tests/conftest.py

```diff
@@ -106,3 +108,21 @@ def teardown():
 
     request.addfinalizer(teardown)
     return app
+
+
+@pytest.fixture()
+def communities_creation(app):
+    """Create some example communities."""
+    with app.app_context():
+        user1 = create_test_user()
```

Comment:
The fixture can be split into 1) test user creation and 2) community creation (which can depend on test user(s))